src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -0,0 +1,300 @@
+              );
+
+      rightClimberArmSim =
+          new SingleJointedArmSim(
+              ClimberConstants.CLIMBER_MECH_MOTORS,
+              ClimberConstants.GEAR_RATIO, // double gearing0,
+              ClimberConstants.CLIMBER_MOI, // double jKgMetersSquared,
+              ClimberConstants.CLIMBER_LENGTH, // double armLengthMeters,
+              ClimberConstants.CLIMBER_MIN_ANGLE, // double minAngleRads,
+              ClimberConstants.CLIMBER_MAX_ANGLE, // double maxAngleRads,
+              true, // boolean simulateGravity,
+              0 // double startingAngleRads
+              );
+
+      leftClimberMech = new Mechanism2d(2, 2);
+      leftMechRoot = leftClimberMech.getRoot("Climber Left Arm Root", 1, 0);
+      leftMechBase =
+          leftMechRoot.append(
+              new MechanismLigament2d(
+                  "Climber Left Base Lig", ClimberConstants.CLIMBER_BASE_LENGTH, 90));
+      leftMechArm =
+          leftMechBase.append(
+              new MechanismLigament2d(
+                  "Climber Left Arm Lig",
+                  ClimberConstants.CLIMBER_LENGTH,
+                  leftClimberArmSim.getAngleRads()));
+
+      rightClimberMech = new Mechanism2d(2, 2);
+      rightMechRoot = rightClimberMech.getRoot("Climber Right Arm Root", 1, 0);
+      rightMechBase =
+          rightMechRoot.append(
+              new MechanismLigament2d(
+                  "Climber Right Base Lig", ClimberConstants.CLIMBER_BASE_LENGTH, 90));
+      rightMechArm =
+          rightMechBase.append(
+              new MechanismLigament2d(
+                  "Climber Right Arm Lig",
+                  ClimberConstants.CLIMBER_LENGTH,
+                  rightClimberArmSim.getAngleRads()));
+
+      ClimberTab.add("Left Climber Mech", leftClimberMech);
+      ClimberTab.add("Right Climber Mech", rightClimberMech);
+    }
+
+    leftClimbPosition =
+        new DebugEntry<Double>(leftEncoder.getPosition(), "Current right climb position", this);
+    rightClimbPosition =
+        new DebugEntry<Double>(rightEncoder.getPosition(), "Current left climb position", this);
+    goalPosition =
+        new DebugEntry<Double>(
+            climbStates.IDLE_SETPOINT.getStateAngle(), "Current Climber Position Goal", this);
+    goalName =
+        new DebugEntry<String>(climbStates.IDLE_SETPOINT.name(), "Current Climber State", this);
+    motorGoalEntry =
+        new DebugEntry<Double>(
+            climbStates.IDLE_SETPOINT.getMotorAngle(), "Current Motor Goal", this);
+    leftMotorOutput =
+        new DebugEntry<Double>(leftMotorController.get(), "Left Climber Output", this);
+    rightMotorOutput =
+        new DebugEntry<Double>(rightMotorController.get(), "Right Climber Output", this);
+  }
+
+  public void applyDemand() {
+    leftMotorController.setVoltage(ClimberConstants.CLIP_VOLTAGE);
+    rightMotorController.setVoltage(ClimberConstants.CLIP_VOLTAGE);
+  }
+
+  public void stopMotors() {
+    leftMotorController.setVoltage(0);
```

Comment:
Is this change for deploying or for simulation, or both?

Reply:
We could do an "isSimulation" if statement here if we want stopMotor for the real